src/test.rs

```diff
@@ -50,6 +50,51 @@ mod tests {
         builder.connect("goggle.com", s).unwrap();
     }
 
+    #[test]
+    fn connect_no_root_certs() {
```

Comment:
Let's add another assertion that we **can** connect to `google.com` in the default case -- just in case the test suite setup changes to not include those certs in the built in certs.

We should also add a separate test to check that we can actually connect when we supply the right cert.

Reply:
That already exists: https://github.com/sfackler/rust-native-tls/blob/3a29e956c1eadba0364b3071569aa71f9defc1ff/src/test.rs#L23

Reply:
>We should also add a separate test to check that we can actually connect when we supply the right cert.

I'm not sure we want to hardcode the CA cert for google.com since it'll change over time? That's the point of the `server_no_root_certs` test.

Reply:
Cool, as long as we're confident that we'll never remove `connect_google`, this is fine.